yarn-project/aztec.js/src/wallet/base_wallet.ts

```diff
@@ -118,4 +118,7 @@ export abstract class BaseWallet implements Wallet {
   addAuthWitness(authWitness: AuthWitness) {
     return this.pxe.addAuthWitness(authWitness);
   }
+  cancelPrivateAuthWitness(messageHash: Fr | Buffer): Promise<void> {
```

Comment:
API here is not consistent with above, addAuthWitness vs `cancelPrivateAuthWitness`, 

If above is add, i would assume this should just be `removeAuthWitness`. This is also calling the PXE so private is implied